### PR TITLE
Typography and lists

### PR DESCRIPTION
**Pull request by harshparab**

### CODE REVIEW

**Review by sameep-s — APPROVED** [review]
Looks Good You can Merge.

File: Components/typography.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/navbar.css">
+    <link rel="stylesheet" href="/css/component.css">
+    <link rel="stylesheet" href="/css/sidebar.css">
+    <link rel="stylesheet" href="typography.css">
```
Comment (HrishikeshMahalle) [inline Components/typography.html]: Try to import these to CSS so that you that importing one file does all the job

**Review by HrishikeshMahalle — APPROVED** [review]
Some of the cleanest HTML I've reviewed. Great JOB.

File: Components/typography.html (hunk 1 above)
Comment (harshparab) [reply Components/typography.html]: Done

Comment (harshparab) [reply Components/typography.html]: And thanks for the compliment!

**Pull request merged**